shanoir-ng-datasets/src/main/java/org/shanoir/ng/studycard/model/StudyCardCondition.java

```diff
@@ -14,43 +14,40 @@
 
 package org.shanoir.ng.studycard.model;
 
+import java.util.List;
+
+import javax.persistence.CascadeType;
 import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
 import javax.validation.constraints.NotNull;
 
 import org.hibernate.annotations.GenericGenerator;
 import org.shanoir.ng.shared.core.model.AbstractEntity;
 
-
 @Entity
 @GenericGenerator(name = "IdOrGenerate", strategy = "org.shanoir.ng.shared.model.UseIdOrGenerate")
 public class StudyCardCondition extends AbstractEntity {
 
 	/** UID */
-	private static final long serialVersionUID = 6708188853533591193L;
+	private static final long serialVersionUID = 6708177853555591193L;
 	
-	@NotNull
-	private int dicomTag;
+	private int dicomTagOrField;
 	
-	private String dicomValue;
+	@OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL, orphanRemoval = true)
+	@JoinColumn(name="condition_id")
+	private List<StudyCardConditionValue> values;
```

Comment:
I think you can use @ElementCollection instead

```
@ElementCollection 
private List<String> values;
```

should suffice, from what I see elsewhere in the code (ie MrProtocolMetadata)